internal/manifest/list.go

```diff
@@ -65,6 +69,22 @@ type ListUpdate struct {
 	MediaType string
 }
 
+type instanceOperation int
+
+const (
+	op_ignore instanceOperation = iota + 1
+	op_update
+)
+
+// ListEdit includes the fields which a List's EditInstances() method will modify.
+type ListEdit struct {
+	ListUpdate
+	Platform            *imgspecv1.Platform
+	Schema2ListPlatform *Schema2PlatformSpec
+	Annotations         map[string]string
+	op                  instanceOperation
```

Comment:
Typically I’d expect the operation that determines the semantics, and requirements on, the other fields, to be the first field in a struct.

---

If this is a private field, how is the `copy` caller going to set it? Do you plan to provide specialized constructors for every value of `instanceOperation`? That would work fine but might not be worth the effort for internal-only code.